### ISSUE TITLE
metrics files contains column names with leading blank space

### ISSUE THREAD

**Issue by PythonFZ** [mir-group/nequip#198]
The `*.csv` files contain a blank space before the key value.
E.g. reading them in with pandas requires to add this blank which can be confusing for people not using `wandb`.

```py
df = pd.read_csv("metrics_epoch.csv")
df["training_loss"].plot() # -> raises KeyError
df[" training_loss"].plot() # works
```

Would it be possible to change the header formatting found in the following lines to not contain the leading blank space?
https://github.com/mir-group/nequip/blob/eb6f9bca7b36162abf69ebb017049599b4ddb09c/nequip/train/trainer.py#L1071
https://github.com/mir-group/nequip/blob/eb6f9bca7b36162abf69ebb017049599b4ddb09c/nequip/train/trainer.py#L1079

**Comment by Linux-cpp-lisp** [comment]
Fixed in https://github.com/mir-group/nequip/pull/199/commits/8e8339cd0eb5b89eb5ef1dbb95641022c68e388f

**Comment by Linux-cpp-lisp** [comment]
The fix is merged and released in 0.5.4. Let me know if it resolves your issue!
